components/p3a/managed/remote_config_manager.h

```diff
@@ -3,8 +3,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at https://mozilla.org/MPL/2.0/. */
 
-#ifndef BRAVE_COMPONENTS_P3A_REMOTE_CONFIG_MANAGER_H_
-#define BRAVE_COMPONENTS_P3A_REMOTE_CONFIG_MANAGER_H_
+#ifndef BRAVE_COMPONENTS_P3A_MANAGED_REMOTE_CONFIG_MANAGER_H_
+#define BRAVE_COMPONENTS_P3A_MANAGED_REMOTE_CONFIG_MANAGER_H_
```

Comment:
for future reference, it would be great to split moving files and logic change, if not to different PRs, but at least to distinct commits